src/main/webapp/i18n/de/messages.json

```diff
@@ -0,0 +1,20 @@
+{
+    "artemisApp": {
+        "messages": {
+            "label": "Nachrichten",
+            "conversation": {
+                "header": "Gespräch",
+                "created": "Gespräch erstellt",
+                "show": "Gespräch anzeigen",
+                "hide": "Gespräch ausblenden",
+                "notFound": "Kein Gespräch gefunden",
```

Comment:
When applying the previous comment, this would change to "Keine Konversation gefunden"

Reply:
👍🏻